integration-tests/cli/csv-parser-orders.it.js

```diff
@@ -14,161 +14,87 @@ describe('CSV and CLI Tests', () => {
     })
 
-    it('should print the module version given the version flag', done => {
-      exec(`${binPath} --version`, (error, stdout, stderr) => {
-        expect(stdout).toBe(`${version}\n`)
-        expect(error && stderr).toBeFalsy()
-        done()
-      })
+    it('should print the module version given the version flag', async () => {
+      const [stdout, stderr] = await exec(`${binPath} --version`)
+      expect(stderr).toBeFalsy()
+      expect(stdout).toBe(`${version}\n`)
     })
 
-    it('should write output to file', done => {
+    it('should write output to file', async () => {
       const csvFilePath = path.join(samplesFolder, 'return-info-sample.csv')
       const jsonFilePath = tmp.fileSync().name
-      const expectedResult = [
-        {
-          orderNumber: '123',
-          returnInfo: [
-            {
-              returnTrackingId: 'aefa34fe',
-              _returnId: '1',
-              returnDate: '2016-11-01T08:01:19+0000',
-              items: [
-                {
-                  quantity: 4,
-                  lineItemId: '12ae',
-                  comment: 'yeah',
-                  shipmentState: 'Returned',
-                },
-                {
-                  quantity: 4,
-                  lineItemId: '12ae',
-                  comment: 'yeah',
-                  shipmentState: 'Returned',
-                },
-              ],
-            },
-            {
-              returnTrackingId: 'aefa34fe',
-              _returnId: '2',
-              returnDate: '2016-11-01T08:01:19+0000',
-              items: [
-                {
-                  quantity: 4,
-                  lineItemId: '12ae',
-                  comment: 'yeah',
-                  shipmentState: 'Unusable',
-                },
-              ],
-            },
-          ],
-        },
-        {
-          orderNumber: '124',
-          returnInfo: [
-            {
-              returnTrackingId: 'aefa34fe',
-              _returnId: '2',
-              returnDate: '2016-11-01T08:01:19+0000',
-              items: [
-                {
-                  quantity: 4,
-                  lineItemId: '12ae',
-                  comment: 'yeah',
-                  shipmentState: 'Unusable',
-                },
-              ],
-            },
-          ],
-        },
-      ]
-
-      exec(
-        `${binPath} -i ${csvFilePath} -o ${jsonFilePath} -t returninfo`,
-        (cliError, stdout, stderr) => {
-          expect(cliError && stderr).toBeFalsy()
 
-          fs.readFile(jsonFilePath, { encoding: 'utf8' }, (error, data) => {
-            expect(error).toBeFalsy()
-            expect(JSON.parse(data)).toEqual(expectedResult)
-            done()
-          })
-        }
+      await exec(
+        `${binPath} -i ${csvFilePath} -o ${jsonFilePath} -t returninfo`
       )
+      const data = await fs.readFile(jsonFilePath, { encoding: 'utf8' })
+      expect(JSON.parse(data)).toMatchSnapshot()
```

Comment:
You don't need to parse it to json. You can simply match it with the string

Reply:
Good find, fixed.